src/App.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { HashRouter as Router, Switch, Route } from "react-router-dom";
```

Comment:
we need to change to HashRouter here because GitHub pages doesn't work with BrowserRoute